app/controllers/katello/api/v2/simple_content_access_controller.rb

```diff
@@ -1,42 +1,29 @@
 module Katello
   class Api::V2::SimpleContentAccessController < Api::V2::ApiController
-    before_action :find_organization
-
     resource_description do
       description "Red Hat subscriptions management platform."
       api_version 'v2'
     end
 
-    api :GET, "/organizations/:organization_id/simple_content_access/eligible",
-      N_("Check if the specified organization is eligible for Simple Content Access. %s") % sca_only_deprecation_text, deprecated: true
+    def render_sca_410_error
+      render_error 'custom_error', status: :gone,
+                                    locals: { message: N_('Simple Content Access is the only supported content access mode') }
```

Comment:
Do you think we should add onto the message saying entitlement mode has been removed, or do you think that covers it. You are better with words than me 😛 so will leave it up to you.

Reply:
I feel like this is a question of "highlight the change" vs. "highlight the way it is now." And I think the proper place for "highlight the change" is in release notes, blog posts, etc. Whereas with the API response, to me it makes more sense to "highlight the way it is now," especially since you never know, this error may be around long enough that the change is a distant memory :smile:

Reply:
Works for me, true communication in the form of marketing and release notes/docs is the better way to explain this in much more depth to the customer. Resolving this topic.